lib/mix/tasks/appsignal.demo.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Mix.Tasks.Appsignal.Demo do
+  use Mix.Task
+
+  @shortdoc "Perform and send a demonstration error and performance issue to AppSignal."
+
+  def run(_args) do
+    {:ok, _} = Application.ensure_all_started(:appsignal)
+    Appsignal.Demo.create_transaction_performance_request
+    Appsignal.Demo.create_transaction_error_request
+    Appsignal.stop(nil)
+  end
```

Comment:
Maybe it's nice to give some feedback to the user to tell that there were 2 samples submitted and they can now open Appsignal to view them?